main.go

```diff
@@ -233,8 +233,19 @@ func connectDb() (atcDb.Conn, lock.LockFactory, error) {
 		return nil, nil, err
 	}
 
+	var strategy encryption.Strategy
```

Comment:
`strategy = encryption.NewNoEncryption()` could be the default value of `strategy` to remove the `else` condition.
```suggestion
	var strategy encryption.Strategy = encryption.NewNoEncryption()
```

Reply:
fixed